code/modules/mob/living/simple_animal/aliens/statue.dm

```diff
@@ -0,0 +1,423 @@
+	icon = 'icons/obj/statue.dmi'
+	tt_desc = "angelum weepicus"
+	icon_state = "human_male"
+	icon_living = "human_male"
+	icon_dead = "human_male"
+	intelligence_level = SA_HUMANOID
+	var/annoyance = 30 //stop staring you creep
+	var/respond = 1
+	var/banishable = 0
+	faction = "statue"
+	mob_size = MOB_HUGE
+	response_help = "touches"
+	response_disarm = "pushes"
+	environment_smash = 2
+	can_be_antagged = 1
+	speed = -1
+	maxHealth = 50000
+	health = 50000
+//	investigates = 1
+
+
+	harm_intent_damage = 60
+	melee_damage_lower = 50
+	melee_damage_upper = 70
+	attacktext = "clawed"
+	attack_sound = 'sound/hallucinations/growl1.ogg'
+
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
+	maxbodytemp = 9000
+	run_at_them = 0
+
+
+	move_to_delay = 0 // Very fast
+
+	animate_movement = NO_STEPS // Do not animate movement, you jump around as you're a scary statue.
+
+	see_in_dark = 13
+	view_range = 35 //So it can run at the victim when out of the view
+
+	melee_miss_chance = 0
+
+
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+	sight = SEE_SELF|SEE_MOBS|SEE_OBJS|SEE_TURFS
+	anchored = 1
+	var/last_hit = 0
+	var/cannot_be_seen = 1
+	var/mob/living/creator = null
+	mob_swap_flags = null
+
+// No movement while seen code.
+
+/mob/living/simple_animal/hostile/statue/New(loc)
+	..()
+	// Give spells
+	add_spell(new/spell/aoe_turf/flicker_lights)
+	add_spell(new/spell/aoe_turf/blindness)
+	add_spell(new/spell/aoe_turf/shatter)
+
+/mob/living/simple_animal/hostile/statue/DestroySurroundings()
+	if(can_be_seen(get_turf(loc)))
+		if(client)
+			to_chat(src, "<span class='warning'>You cannot move, there are eyes on you!</span>")
+		return 0
+	return ..()
+
+/mob/living/simple_animal/hostile/statue/attackby(var/obj/item/O as obj, var/mob/user as mob)
+	if(istype(O, /obj/item/weapon/nullrod))
+		visible_message("<span class='warning'>[user] tries to banish [src] with [O]!</span>")
+		if(do_after(user, 15, src))
+			if(banishable)
+				visible_message("<span class='warning'>[src] crumbles into dust!</span>")
+				gib()
+			else
+				visible_message("<span class='warning'>[src] is too strong to be banished!</span>")
+				Paralyse(rand(8,15))
+
+/mob/living/simple_animal/hostile/statue/death()
+	new /obj/item/stack/material/marble(loc)
+
+/mob/living/simple_animal/hostile/statue/Move(turf/NewLoc)
+	if(can_be_seen(NewLoc))
+		if(client)
+			to_chat(src, "<span class='warning'>You cannot move, there are eyes on you!</span>")
+		return 0
+	return ..()
+
+/mob/living/simple_animal/hostile/statue/Life()
+	..()
+	handleAnnoyance()
+	if(target_mob)
+		if((annoyance + 4) < 800)
+			annoyance += 4
+	else if ((annoyance - 2) > 0)
+		annoyance -= 2
+
+/mob/living/simple_animal/hostile/statue/handle_stance()
+	if(!..())
+		return
+	if(target_mob) // If we have a target and we're AI controlled
+		var/mob/watching = can_be_seen()
+		// If they're not our target
+		if(watching && watching != target_mob)
```

Comment:
`watching && watching`

Reply:
Doesn't watching != target_mob return true if watching == null?
In this case if someone got blinded, but still a target, they will stop being one, making statue ignore literally everyone.
Or do you want me to add brackets to the sides?